srcpkgs/taplo/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'taplo'
+pkgname=taplo
+version=0.7.0
+revision=1
+wrksrc="${pkgname}-release-${pkgname}-cli-${version}"
+build_wrksrc="crates/taplo-cli"
+build_style=cargo
+hostmakedepends="pkg-config"
+makedepends="openssl-devel"
+short_desc="TOML toolkit written in Rust"
```

Comment:
"TOML toolkit"?

Reply:
I took it from the repo description. I use it for automatically formatting TOML files with helix.